python/orca/tutorial/xshards/notebooks/ionosphere.ipynb

```diff
@@ -428,8 +428,8 @@
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "\r",
-      "[Stage 12:>                                                         (0 + 1) / 1]\r",
+      "\r\n",
+      "[Stage 12:>                                                         (0 + 1) / 1]\r\n",
```

Comment:
revert this use terminal editor `vim` or `nano`, vscode sometimes adds `\n` out of no reason 😭